pkg/ip/ip.go

```diff
@@ -764,6 +764,19 @@ func KeepUniqueIPs(ips []net.IP) []net.IP {
 	return returnIPs
 }
 
+func KeepUniqueAddrs(addrs []netip.Addr) []netip.Addr {
```

Comment:
Do we want to perform a copy first then sort?

If not, we should comment this so the caller knows their original `addrs` array will be modified (sorted, in this case).

Alternatively, you can make this function take `addrs *[]netip.Addr` and return no value, which indicates to the caller the array is modified completely in place. The returned slice is always < len(addrs) so I believe you won't strictly need to return `[]netip.Addr` 

If the function receives a pointer to a slice you can then update the pointer's value to the new slice 

For example line 769 becomes
```
*addrs := (*addrs)[:0]
```

Reply:
I've tried to keep this as close as possible to `KeepUniqueIPs` which modifies the original `addrs` slice. The way that function is currently used, the in-place modification and sorting is expected. Thus, I'd keep it as is for now and add a godoc comment describing its behavior.

Reply:
lgtm